src/composables/Dialog.js

```diff
@@ -0,0 +1,44 @@
+import { onMounted, onUnmounted, ref } from 'vue';
+import DialogEvent from 'src/composables/DialogEvent';
+
+/**
+ * Composable that setup specific dialog referenced by key.
```

Comment:
Composable that sets up specific dialog referenced by key.

Reply:
done